contracts/HATVaultsRegistry.sol

```diff
@@ -79,9 +77,8 @@ contract HATVaultsRegistry is Ownable {
     struct SwapData {
```

Comment:
i know it's too late but i don't get why to create this struct if all it's members are just internal function variables and not passed anywhere.. makes more sense to me to just have them as local variables in the one function where you use it.. but it's not important so feel free to ignore :)

Reply:
Because this allows to overcome the stack too deep error

Reply:
there's a comment in the code that is it to avoid stack too deep erros

Reply:
ohhh ok missed that sorry